src/test/java/com/github/algobot76/surabaya/AnalyzerTests.java

```diff
@@ -0,0 +1,95 @@
+package com.github.algobot76.surabaya;
```

Comment:
Please consider moving `AnalyzerTests.java` to `src/test/java/com/github/surabaya/util` directory.